tasks/functions/functions.js

```diff
@@ -13,34 +14,61 @@ function add(a, b) {
  *    firstName: "John"
  *    lastName: "Dou"
  * }
+ *
  */
-function getFullName(object) {
-
-}
-
+/*var person = {
+    firstName: "John",
+    lastName : "Dou"};
+    */
+   function getFullName(firstName, lastName) {
+    var obj = {
+      firstName: firstName,
+      lastName: lastName
+  };
+    return(obj.firstName + " " + obj.lastName);
+  }
+  getFullName("John", "Dou");
 /**
  * write fuction that checks is number is odd
  * true if odd, false if even
  */
 function isOdd(n) {
-
-}
+    if (n%2 == 1) {
+  return true;
+    } else {
+    return false;
+    }
+};
+isOdd(2);
+isOdd(3);
+isOdd(0);
 
 /**
  * write function that return shortest of words in given array
  * e.g ["one", "two", "three"] should return one
  */
-function getShortest(wordArray) {
+var wordArray = ["one", "two", "three"];
 
-}
+function getShortest(arr){
+  return (
+  arr.reduce((a, b) => a.length <= b.length ? a : b)
```

Comment:
it's a good thing to use reduce, +1 point from me :)

But let's also try using for loop in your fee time :)

Reply:
remained as is